api/urls.py

```diff
@@ -94,5 +94,6 @@ def register_url(self, subclass, viewset):
     url(r'search/(?!.*_bulk\/?$)(?P<url_bits>.*)', csrf_exempt(views.ElasticSearchView.as_view()), name='search'),
 
     url(r'schema/?$', views.SchemaView.as_view(), name='schema'),
+    url(r'modeltypes/?$', views.ModelTypesView.as_view(), name='modeltypes'),
```

Comment:
This should be somewhere a bit more meaningful. Maybe just a generic `hierarchy/<modelname>` or `schema/<modelname>/types|hierarchy`?